docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,267 @@
+
+### UID2 Operator Account Setup
+Ask your UID2 contact to register your organization as a UID2 Operator. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+(**GWH_YS03 what information do they need to provide for registration? In case you don't know, who should I ask?**)
+
+When the registration process is complete, you'll receive the following:
+
+ - `{IMAGE_SHA}`: A valid UID2 operator image digest, used in configuration. For details, see [Integration Deployment](#integration-deployment) and  [Production Deployment](#production-deployment).
+ - `{API_TOKEN}`: An API token, exclusive to you, that identifies you with the UID2 service as a private operator. This value is used in configuration. For details, see [Integration Deployment](#integration-deployment) and  [Production Deployment](#production-deployment).
+
+(**GWH_YS04 what exactly do we send them? Is there more than the above? GEN TO REVIEW THE RECORDING BEFORE ASKING THIS ALSO: do they use the same value for integ and prod? Or do they get separate values? We should be clear about that**)
+
+When registration is complete, and your Confidential Space account is set up, it's time to download the Docker build and set up your GPC Confidential Space enclave.
+
+
+## Build
+
+To run UID2 Operator on a GCP Confidential Space enclave, download the official Docker image from the following Google Container Registry location:
+
+- [ghcr.io/iabtechlab/uid2-operator](ghcr.io/iabtechlab/uid2-operator) (**GWH_YS05 this redirects to https://github.com/iabtechlab/uid2-operator/pkgs/container/uid2-operator. Should I use that?**)
+
+For example:
+
+```
+docker pull ghcr.io/iabtechlab/uid2-operator
+```
+
+You can use the following command to build a non-certified UID2 operator container image from the source code:
+
+```
+# From the root source folder
+# Update project version in pom to "1.0.0-SNAPSHOT"
+
+mvn -B package -P gcp 
+cp -r target scripts/gcp-oidc/
+docker build ./scripts/gcp-oidc/. -t ghcr.io/iabtechlab/uid2-operator:v1.0.0-SNAPSHOT
+```
+## Deploy
+
+(**GWH_YS06 is there something re can say re when they deploy integ versus production? Do they do integ then production? Do they generally have both, or transition one to the other? Some general info would be helpful.**)
+
+The following environments are available:
+
+- Integration environment: see [Integration Deployment](#integration-deployment)
+- Production environment: see [Production Deployment](#production-deployment)
+
+### Integration Deployment
+
+To deploy a new UID2 Operator in the GCP Confidential Space Enclave into the integration environment, follow these steps.
+
+#### (For uid2 admin) Register Enclave ID in the Admin Portal
+
+1. Log in to the Admin Portal.
+
+2.  Go to the [GCP Enclave ID page](https://admin-integ.uidapi.com/adm/enclave-gcp-v2.html) and provide the following values to generate the Enclave ID: (**GWH_YS07 I cannot access this URL. Sure it's valid?**)
```

Comment:
We could skip the whole section (for uid2 admin). It does not provide any meaningful info for partners/ public viewers.

The link here is internal and has some sort of authentication. ( It's for uid2 admin. )

Reply:
Done, thx.